documentation/modules/post/windows/gather/credentials/teamviewer_passwords.rb

```diff
@@ -0,0 +1,26 @@
+## Vulnerable Application
```

Comment:
The file should use the `.md` extension because it's Markdown not Ruby.

Reply:
Yes, that is my mistake ...something something copy paste...